bin/sdk/notifications.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * External dependencies
+ */
+const HtmlWebpackPlugin = require( 'html-webpack-plugin' );
+const path = require( 'path' );
+const spawnSync = require( 'child_process' ).spawnSync;
+
+exports.config = ( { argv: { outputDir }, getBaseConfig, __rootDir } ) => {
+	const baseConfig = getBaseConfig();
+
+	return {
+		...baseConfig,
+		...{
+			entry: path.join( __rootDir, 'client', 'notifications', 'src', 'standalone' ),
+			node: {
+				fs: 'empty',
+			},
+			output: {
+				path: outputDir,
+				filename: 'build.min.js',
+			},
+			plugins: [
+				...baseConfig.plugins,
+				new HtmlWebpackPlugin( {
+					filename: path.join( outputDir, 'root.html' ),
+					gitDescribe: spawnSync( 'git', [ 'describe', '--always', '--dirty', '--long' ], {
```

Comment:
TIL about `git describe`! Will it do anything sensible though in the Calypso repo? Right now it shows `desktop/2.4.0-12909-ga8ca144600`

Reply:
we added this in after several issues related to the build environment. at worst it's an extra meta tag but at best it helps us quickly pinpoint variables that could have caused a broken build